test/create/enableExisting.test.ts

```diff
@@ -18,38 +18,38 @@ sqs.mockImplementationOnce(() => ({ getQueueUrl }))
 import { enableExisting as ee } from "../../src/create/enableExisting"
```

Comment:
Not a change you made, but--

could this be `enableExisting` without the alias?

Reply:
Good catch! Updated